src/commands/shared/ssh.ts

```diff
@@ -131,6 +131,7 @@ export const provisionRequest = async (
   }
   const { id, isPreexisting } = response;
   if (!isPreexisting) print2("Waiting for access to be provisioned");
+  else print2("Existing access found.  Connecting to node.");
```

Comment:
We called them `instance` in the CLI help
```suggestion
  else print2("Existing access found.  Connecting to instance.");
```